platform-enterprise_versioned_docs/version-25.1/enterprise/configuration/overview.mdx

```diff
@@ -135,7 +135,7 @@ Configuration values that control Seqera's interaction with databases and Redis
 From Seqera Enterprise version 24.2:
 
 - Redis version 6.2 or greater is required.
-- Redis version 7 is officially supported.
+- Redis version 7 and greater is officially supported.
```

Comment:
Saying "and greater" is misleading (might make folks this version 8 is supported, which it isn't yet).

```suggestion
- Redis version 7.X is officially supported
```